src/data/markdown/docs/01 guides/05 Testing Guides/04 Running large tests.md

```diff
@@ -45,7 +42,9 @@ ulimit -n 250000
 
 </div>
 
-For quick testing, you can paste these commands in a terminal window as the root user. To make these changes permanent, refer to the instructions of your Linux distribution.
+These commands enable reusing network connections, increase the limit of network connections, and range of local ports. 
+
```

Comment:
I think we should add this sentence:

To apply these changes, you can either paste these commands as a root user before running a k6 test or change the configuration files in your operating system.

Reply:
https://github.com/loadimpact/k6-docs/pull/25/commits/6f03166368b5b118f89fc4805d93c533c98d48a4